src/StateT.ts

```diff
@@ -130,6 +134,7 @@ export function getStateM<M>(M: Monad<M>): StateM<M> {
     gets: f => s => M.of([f(s), s]),
     fromState: sa => s => M.of(sa(s)),
     fromM: ma => s => M.map(ma, a => [a, s]),
+    runState: (ma, s) => M.map(ma(s), ([a, s]) => [a, s]),
```

Comment:
This could implemented instead as
```
runState: (ma, s) => M.map(ma(s), x => x),
```
or
```
import { identity } from '../src/function'
...
runState: (ma, s) => M.map(ma(s), identity),
```

Think that'd be better?